Source/WebCore/dom/PointerEvent.idl

```diff
@@ -32,6 +32,8 @@ dictionary PointerEventInit : MouseEventInit {
     long tiltX = 0;
     long tiltY = 0;
     long twist = 0;
+    double altitudeAngle;
+    double azimuthAngle;
```

Comment:
If the pointer is a penPointerEventType() then a device that supports tilt (like the Apple Pencil) is being used so it can report a calculated value, if it is a different pointer then tilt isn't supported in which case it returns the default value.

Reply:
You probably meant to put this reply in the other thread!

Reply:
You ought to still gate those dict values behind `[EnabledBySetting=...]`?